src/conversions/lanelet.cpp

```diff
@@ -135,7 +217,8 @@ std::vector<LanePoint> LaneletConverter::fromPolygon(
       dy /= (norm + epsilon);
       dz /= (norm + epsilon);
     }
-    output.emplace_back(itr->x(), itr->y(), itr->z(), dx, dy, dz, 0.0);  // TODO(ktro2828): Label ID
+    output.emplace_back(
+      itr->x(), itr->y(), itr->z(), dx, dy, dz, label_id);  // TODO(ktro2828): Label ID
```

Comment:
[NITS] TODO can be removed.

```suggestion
      itr->x(), itr->y(), itr->z(), dx, dy, dz, label_id);
```